CMP1903_A1_2324/Game.cs

```diff
@@ -6,16 +6,46 @@
 
 namespace CMP1903_A1_2324
 {
-    internal class Game
+    public class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
 
-        //Methods
+        
+        //Sum and report the total of the three dice rolls.
+
+        //when creating an instance of a class (a new object), you have to write the class name so that
+        //it knows which class to construct from
+        //Creates three new dice objects
+        public Die Die1 = new Die();
+        public Die Die2 = new Die();
+        public Die Die3 = new Die();
+
```

Comment:
Using the Google C# code style to annotate that Die1, 2 and 3 are objects, not variables